transformation/src/jade-schema/tables/alignment_file.table.json

```diff
@@ -0,0 +1,142 @@
+{
+  "name": "alignment_file",
+  "columns": [
+    {
+      "name": "id",
+      "datatype": "string",
+      "type": "primary_key"
+    },
+    {
+      "name": "byte_size",
+      "datatype": "integer",
+      "type": "required"
+    },
+    {
+      "name": "time_created",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "data_modality",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "audit_label",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "audit_severity",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "award",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "file_format_type",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "lab",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "platform",
+      "datatype": "string"
+    },
+    {
+      "name": "quality_metrics",
+      "datatype": "string",
+      "type": "repeated"
+    },
+    {
+      "name": "restriction_enzymes",
+      "datatype": "string",
+      "type": "repeated"
+    },
+    {
+      "name": "status",
```

Comment:
We only intend to store `released` things, so I don't think we need this on any object

Reply:
👍  I just removed the status column.